SLRemoting/SLObject.m

```diff
@@ -229,7 +229,7 @@ + (NSDictionary *)convertArguments:(NSDictionary *)prop {
         id value = [prop objectForKey:key];
         if ([value isKindOfClass:[NSDate class]]) {
             NSString *jsonString = [value convertToJSONString];
-            value = @{ @"$type": @"date", @"$data": jsonString };
+            value = jsonString ;
```

Comment:
> I think this is a subtle but also an important breaking change, could you please add a note to both Release Notes and Migration Guide for 3.0?

I mean that Release Notes should mention that LoopBack 3.x not longer recognizes nested query arguments encoded using `{$type,$data}` schema and show what's the new format. The Migration Guide should have a short section telling users to upgrade their client apps.